velox/tool/trace/OperatorReplayerBase.cpp

```diff
@@ -103,4 +108,15 @@ OperatorReplayerBase::replayNodeFactory(const core::PlanNode* node) const {
     return createPlanNode(node, nodeId, source);
   };
 }
+
+void OperatorReplayerBase::printOperatorStats(
```

Comment:
```
s/printOperatorStats/printStats/
```
Could you also print out the tree memory usage by calling treeMemoryUsage without skipping empty pool? thanks!